cleverhans/picklable_model.py

```diff
@@ -575,3 +575,114 @@ def fprop(self, x, dropout=False, dropout_dict=None, **kwargs):
         if dropout:
             return tf.nn.dropout(x, include_prob)
         return x
+
+
+class Residual(Layer):
```

Comment:
I guess there can be different residual layers as suggested by the comment below. Would it make sense to name this `ResidualWithGroupNormalization` in case we want to implement other residual network layers or does that sound too long?